### ISSUE TITLE
Support for Safari

### ISSUE THREAD

**Issue by Salil999** [maxtheaxe/wordle-sync#4]
Would it be possible to have syncing support for Safari?

**Comment by maxtheaxe** [comment]
It's possible, yes, but the process is a little more involved, because Apple doesn't _actually_ support [storage.sync](https://developer.apple.com/documentation/safariservices/safari_web_extensions/assessing_your_safari_web_extension_s_browser_compatibility), the browser syncing API that I use in the firefox and chrome versions (see me whining about it [here](https://twitter.com/MaxPerrello/status/1491113518120448001)). Basically, I'd need to build my own syncing service, which comes with some other problems. I had originally intended to do so, but never got around to it. I'd still love to at some point, because that would allow proper mobile support (which was the original intended use-case, anyway), but if and when that happens remains to be seen.

If that's something you'd like to help build, I'd be happy to chat about it and we can see what that might entail.

**Comment by Salil999** [comment]
Yeah I would like to help. It kinda aligns with #3 - I would love to help create a backend for this.

Maybe a username / password approach?

**Comment by maxtheaxe** [comment]
#3 was intended to be for things like [the wordle archive](https://www.devangthakkar.com/wordle_archive/) (basically external hosts for the same game, in case the NYT version became unusable), though safari support was also independently on the "list" (just forgot to track it as an issue).

My inclination has always been to avoid the responsibility of storing peoples' data, so any way to minimize that would be optimal. To that end, my initial thought was allowing OAuth only (potentially using something like [Auth0](https://auth0.com/)), but I honestly haven't built anything before that requires creation of user accounts, so I'll happily defer to your better judgement.
